MailCore/Utils/MailTextStyle.swift

```diff
@@ -196,4 +196,9 @@ public extension MailTextStyle {
         mailFont: .label,
         color: MailResourcesAsset.redColor.swiftUIColor
     )
+
+    static let labelDraggableThread = MailTextStyle(
+        font: .system(size: UIFontMetrics.default.scaledValue(for: 36)),
+        color: .accentColor
+    )
```

Comment:
Creating a MailTextStyle here is not necessary. MailTextStyle lists all the styles used (and re-used) across the views of the app: body, title, caption…

`labelDraggableThread` is used only once. You can declare the font directly where you need it 👍